wagtail/admin/templates/wagtailadmin/shared/buttons.stories.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
```

Comment:
I recommend you call this file `button.stories.tsx` (singular) as this will align with the button template tag that will likely be created by other work.

Reply:
Sorry for the confusion - we may not do the template tag yet - I meant just change the file name to be singular not plural.